src/w3d/renderer/motchan.cpp

```diff
@@ -385,11 +385,11 @@ bool TimeCodedBitChannelClass::Load_W3D(ChunkLoadClass &cload)
     m_pivotIdx = chan.Pivot;
     m_defaultVal = chan.DefaultVal;
     m_cachedIdx = 0;
-    int bytesleft = 4 * m_numTimeCodes - 4;
+    const int bytesleft = 4 * m_numTimeCodes - 4;
 
-    captainslog_assert((sizeof(chan) + bytesleft) == (unsigned)chunk_size);
+    captainslog_assert((sizeof(chan) + bytesleft) == chunk_size);
 
-    m_bits = new unsigned long[m_numTimeCodes];
+    m_bits = new unsigned int[m_numTimeCodes];
```

Comment:
Confirmed as being unsigned long in the original.

Reply:
strongly suggest changing all these to `uint64_t` as then its much easier to read.

Reply:
`long` is 32 bit though.

Reply:
https://github.com/TheAssemblyArmada/Thyme/wiki/Thyme-Style-Guide also as per the style guide we should be avoiding it

Reply:
I am ok with making it `uint32_t`